apps/login/src/components/session-item.tsx

```diff
@@ -141,7 +141,7 @@ export function SessionItem({ session, reload, requestId }: { session: Session;
 
             <XCircleIcon
               className="h-5 w-5 opacity-50 transition-all hover:opacity-100 group-hover:block sm:hidden"
-              onClick={(event) => {
+              onClick={(event: React.MouseEvent) => {
```

Comment:
why the explicit type? Does using async work?

```jsx
            <XCircleIcon
              className="h-5 w-5 opacity-50 transition-all hover:opacity-100 group-hover:block sm:hidden"
              onClick={async (event) => {
                event.preventDefault();
                event.stopPropagation();
                await clearSessionId(session.id);
                reload();
              }}
            />
```

Reply:
async does work, the type is not strictly required. I've changed it